src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactDetailsHeader.tsx

```diff
@@ -106,19 +100,19 @@ export const ContactDetailsHeader: React.FC<Props> = ({
                 }}
               />
             </Box>
-          ) : data.contact ? (
+          ) : (
             <>
               <PrimaryContactName data-testid="ContactName" variant="h5">
-                {data.contact.name}
+                {data?.contact.name}
               </PrimaryContactName>
               <IconButton
```

Comment:
Should we make this button a little bigger? 24px like the others would probably be great.